apps/marketing/src/app/(marketing)/open/monthly-total-users-chart.tsx

```diff
@@ -14,7 +14,7 @@ export type MonthlyTotalUsersChartProps = {
 export const MonthlyTotalUsersChart = ({ className, data }: MonthlyTotalUsersChartProps) => {
   const formattedData = [...data].reverse().map(({ month, cume_count: count }) => {
     return {
-      month: DateTime.fromFormat(month, 'yyyy-MM').toFormat('LLL'),
+      month: DateTime.fromFormat(month, 'yyyy-MM').toFormat('LLLL'),
```

Comment:
Same ting for all of these

Reply:
I thought instead of just showing it as Mar Feb, why don't we show it completely as March or February 😶